crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -17,10 +17,16 @@ use bevy_render::{
 #[bind_group_data(StandardMaterialKey)]
 #[uniform(0, StandardMaterialUniform)]
 pub struct StandardMaterial {
+    /// The color of the surface of the material before lighting.
+    ///
     /// Doubles as diffuse albedo for non-metallic, specular for metallic and a mix for everything
     /// in between. If used together with a base_color_texture, this is factored into the final
     /// base color as `base_color * base_color_texture_value`
     pub base_color: Color,
+
+    /// The [`base_color`] as a texture.
```

Comment:
I think this contradicts the above comment. This seems to imply this is exclusive, while the above says that if both are present they're composed together (multiplied). Can we clarify? For example:

```suggestion
    /// A texture used to modulate the [`base_color`].
```

Reply:
I get what you are getting at. But the idea here was to point the user to the `base_color` doc so that they get the actual explanation without us having to repeat the exact same doc twice.

Note that the documentation for `base_color` and `base_color_texture` will always appear next to each other in the doc pages. Do you have a better suggestion with this in mind?

Also I really dislike "modulate:" it is vague and technical. Good if you are writing doc, but not if you are reading it.

Reply:
The `as` is the issue, see other comment.